kura/org.eclipse.kura.linux.usb.x86_64/META-INF/MANIFEST.MF

```diff
@@ -3,6 +3,6 @@ Bundle-ManifestVersion: 2
 Bundle-Name: Native libraries for 'org.eclipse.kura.linux.usb' on x86_64
 Bundle-SymbolicName: org.eclipse.kura.linux.usb.x86_64
 Bundle-Version: 1.0.0.qualifier
-Fragment-Host: org.eclipse.kura.linux.usb;bundle-version="1.0.8"
+Fragment-Host: org.eclipse.kura.linux.usb;bundle-version="[1.0.8,2.0.0)"
```

Comment:
The above changes should logically go with the previous commit for org.eclipse.kura.linux.usb.